scripts/run.sh

```diff
@@ -64,7 +64,10 @@ echo "Test pod exited with code:${podExitCode}"
 
 #Collect artifacts
 retrive-artifacts
+retrieveArtifactsExitCode=$?
+
 
 #Clean up
 delete_lease
-exit ${podExitCode}
+exitCode=$(${podExitCode}+${retrieveArtifactsExitCode})
```

Comment:
Is there a way we could get both exit codes separately? If both return an exit code, would it be hard to tell what the exit code for each process was?